code/client/munkiimport

```diff
@@ -760,6 +777,8 @@ def main():
 
     parser = PassThroughOptionParser(usage=usage, epilog=epilog)
 
+    parser.add_option('--init', action='store_true',
+                      help='Initialize a munki repository.')
```

Comment:
What is the implied behavior if --init is called against an existing path? "Initialize" sounds like it might clear out the contents. Pretty sure it only creates any missing directories, so the help text might be more explicit.